php-nginx/build-scripts/build_php56.sh

```diff
@@ -128,9 +121,10 @@ mkdir -p ${PHP56_DIR}/lib/conf.d
 
 # Install shared extensions
 ${PHP56_DIR}/bin/pecl install memcache
```

Comment:
It's just installed as shared extension and it's not enabled by default, so I'd keep it.

There are still some code depending on `Memcache` class out there.